### DISCUSSION TITLE
MacOS remux recording progress window gets lost

### DISCUSSION THREAD

**Discussion by XGxF3** [obsproject/obs-studio#10206] [General]
I have "Automatically remux to mp4" on and sometimes the remux recording progress window gets lost behind the main window. This sometimes causes problems because I'll try to quit before the remux is competed. Can the remux recording progress window be made to stay in front of the main window or disable quitting until the remux is competed?

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support or feature request, and we are not currently accepting those requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
